connection.php

```diff
@@ -0,0 +1,725 @@
+<?php
+
+/**
+* @author: mfsi-krishnadev
+* @class : database connection and operation
+* 
+*/
+class db_connection 
+{
+	//declaration of variables
+	private $hostname = "localhost";
```

Comment:
Database configuration should be fetched from a db config file instead of any connection file.